packages/arb-validator-core/message/message.go

```diff
@@ -102,12 +183,14 @@ func UnmarshalFromCheckpoint(msgType MessageType, v value.Value) (InboxMessage,
 		return UnmarshalERC721FromCheckpoint(v)
 	case ContractTransactionType:
 		return UnmarshalContractTransactionFromCheckpoint(v)
+	case TransactionBatchType:
+		return UnmarshalTransactionBatchFromCheckpoint(v)
 	default:
 		return nil, errors.New("bad message type")
 	}
 }
 
-func DeliveredValue(m Message) value.Value {
+func DeliveredValue(m SingleMessage) value.Value {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function DeliveredValue should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)